.github/workflows/ci.yml

```diff
@@ -20,8 +20,26 @@ jobs:
       options: --privileged
     steps:
     - uses: actions/checkout@v2
-    - name: Run storage tests
-      run: ./automation/tests-storage.sh
+    - name: Run storage tests not marked as root
+      run: |
+        ./automation/setup-storage.sh
+        chown -R vdsm:kvm ./ /var/tmp/vdsm*
+        su -c "./automation/tests-storage.sh" -- vdsm
+        make clean-storage
+  test-storage-root:
+    env:
+      TRAVIS_CI: 1
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/ovirt/vdsm-test-centos-8
+      # Required to create loop devices.
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run storage tests marked as root
+      run: |
+        ./automation/setup-storage.sh
+        ./automation/tests-storage.sh
```

Comment:
And this can be:

    ./automation/tests-storage.sh

So automation/tests-storage.sh will prepare the environment and run tox as vdsm if a user was specified.